doc/Support/Poller Support.md

```diff
@@ -42,38 +42,49 @@ These are the default poller config items. You can globally disable a module by
-$config['poller_modules']['ipmi']                         = 1;
-$config['poller_modules']['sensors']                      = 1;
-$config['poller_modules']['processors']                   = 1;
-$config['poller_modules']['mempools']                     = 1;
-$config['poller_modules']['storage']                      = 1;
-$config['poller_modules']['netstats']                     = 1;
-$config['poller_modules']['hr-mib']                       = 1;
-$config['poller_modules']['ucd-mib']                      = 1;
-$config['poller_modules']['ipSystemStats']                = 1;
-$config['poller_modules']['ports']                        = 1;
-$config['poller_modules']['bgp-peers']                    = 1;
-$config['poller_modules']['junose-atm-vp']                = 1;
-$config['poller_modules']['toner']                        = 1;
-$config['poller_modules']['ucd-diskio']                   = 1;
-$config['poller_modules']['wifi']                         = 1;
-$config['poller_modules']['ospf']                         = 1;
-$config['poller_modules']['cisco-ipsec-flow-monitor']     = 1;
-$config['poller_modules']['cisco-remote-access-monitor']  = 1;
-$config['poller_modules']['cisco-cef']                    = 1;
-$config['poller_modules']['cisco-sla']                    = 1;
-$config['poller_modules']['cisco-mac-accounting']         = 1;
-$config['poller_modules']['cipsec-tunnels']               = 1;
-$config['poller_modules']['cisco-ace-loadbalancer']       = 1;
-$config['poller_modules']['cisco-ace-serverfarms']        = 1;
-$config['poller_modules']['netscaler-vsvr']               = 1;
-$config['poller_modules']['aruba-controller']             = 1;
-$config['poller_modules']['entity-physical']              = 1;
-$config['poller_modules']['applications']                 = 1;
-$config['poller_modules']['cisco-asa-firewall']           = 1;
-$config['poller_modules']['mib']                          = 0;
+$config['poller_modules']['unix-agent']                  = 0;
+$config['poller_modules']['os']                          = 1;
+$config['poller_modules']['ipmi']                        = 1;
+$config['poller_modules']['sensors']                     = 1;
+$config['poller_modules']['processors']                  = 1;
+$config['poller_modules']['mempools']                    = 1;
+$config['poller_modules']['storage']                     = 1;
+$config['poller_modules']['netstats']                    = 1;
+$config['poller_modules']['hr-mib']                      = 1;
+$config['poller_modules']['ucd-mib']                     = 1;
+$config['poller_modules']['ipSystemStats']               = 1;
+$config['poller_modules']['ports']                       = 1;
+$config['poller_modules']['bgp-peers']                   = 1;
+$config['poller_modules']['junose-atm-vp']               = 0;
+$config['poller_modules']['toner']                       = 0;
+$config['poller_modules']['ucd-diskio']                  = 1;
+$config['poller_modules']['wifi']                        = 0;
```

Comment:
Can you remove wifi?  It is deprecated.

Reply:
It is still in the defaults file, should it not remain consistent until wifi is obsolete?

Reply:
It's still doing stuff though so should be kept until it is deprecated.

Reply:
No, each OS that needs it has it explicitly enabled in their OS definition.

It is set to false globally by default and there is no reason for a user to change that.